Automattic/jetpack
# Boost: Clear Site Speed Score on plugin deactivation or removal

Fixes #20657 

#### Changes proposed in this Pull Request:
* Fixed a small typo in the class name.
* Added a way to delete all transient added by the `Transient` class by cache prefix.
* Clear all cache for speed score requests on deactivation.

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Generate a speed score in boost and within five minutes complete the remaining steps.
* Deactivate the plugin.
* Activate the plugin again.
* Check to make sure generated speed calculation is removed and it is regenerating the speed score.
